modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/ChannelList.jsx

```diff
@@ -62,4 +64,90 @@ describe('Channels List', () => {
 
 		expect(queryByText('New Property')).toBeNull();
 	});
+
+	it('should check if the number of synced sites and channels appears for each property', async () => {
+		API.channels.search.mockReturnValue(
+			Promise.resolve(data.mockChannels())
+		);
+
+		const {container, getByText} = render(
+			<Provider store={mockStore()}>
+				<MemoryRouter
+					initialEntries={['/workspace/23/settings/properties']}
+				>
+					<Route path={Routes.SETTINGS_CHANNELS}>
+						<ChannelList {...defaultProps} />
+					</Route>
+				</MemoryRouter>
+			</Provider>
+		);
+
+		jest.runAllTimers();
+
+		await waitForLoadingToBeRemoved(container);
+
+		expect(getByText('Sites')).toBeInTheDocument();
+
+		expect(getByText('Channels')).toBeInTheDocument();
+
+		expect(getByText('Liferay DXP')).toBeInTheDocument();
+
+		expect(getByText('6')).toBeInTheDocument();
+
+		expect(getByText('5')).toBeInTheDocument();
+	});
+
+	it('should check if actions "DELETE" and "CLEAR DATA" are displayed in the ellipsis and its icons', async () => {
+		API.channels.search.mockReturnValueOnce(
+			Promise.resolve(data.mockChannels())
+		);
+
+		const {container, getByText} = render(
+			<Provider store={mockStore()}>
+				<MemoryRouter
+					initialEntries={['/workspace/23/settings/properties']}
+				>
+					<Route path={Routes.SETTINGS_CHANNELS}>
+						<ChannelList {...defaultProps} />
+					</Route>
+				</MemoryRouter>
+			</Provider>
+		);
+
+		jest.runAllTimers();
+
+		await waitForLoadingToBeRemoved(container);
+
+		expect(getByText('Delete')).toBeInTheDocument();
+
+		expect(getByText('Clear Data')).toBeInTheDocument();
+
+		const clearDataBtn = screen.getByRole('button', {
+			name: /clear data/i
+		});
+
+		const deleteBtn = screen.getByRole('button', {
+			name: /delete/i
+		});
+
+		expect(clearDataBtn).toBeInTheDocument();
+
+		expect(deleteBtn).toBeInTheDocument();
```

Comment:
Isn't it the same match?

`getByText('Delete')` is equal to `const clearDataBtn = screen.getByRole('button', {
			name: /clear data/i
		});`

Reply:
Remove get by text (2)